curriculum/challenges/english/01-responsive-web-design/css-flexbox/use-the-flex-direction-property-to-make-a-row.english.md

```diff
@@ -8,8 +8,8 @@ forumTopicId: 301110
 
 ## Description
 <section id='description'>
-Adding <code>display: flex</code> to an element turns it into a flex container. This makes it possible to align any children of that element into rows or columns. You do this by adding the <code>flex-direction</code> property to the parent item and setting it to row or column. Creating a row will align the children horizontally, and creating a column will align the children vertically.
-Other options for <code>flex-direction</code> are row-reverse and column-reverse.
+Adding the property <code>display: flex</code> to an element turns the element into a flex container. This makes it possible to align any children of the element into rows or columns. You do this by adding the <code>flex-direction</code> property to the parent item and setting it to <code>row</code> or <code>column</column>. Creating a row will align the children horizontally, while creating a column will align the children vertically.
```

Comment:
`Adding the property <code>display: flex</code> to an element turns the element into a flex container.` creates a tautology.